### ISSUE TITLE
MIPOW help

### ISSUE THREAD

**Issue by amahlaka** [derfloDev/HomeAssistantMipow#1]
Hi, i see you also want to have playbulb support on homeasisstant, i have a working version, i dont know how well it works on more than 1 bulb tho

**Comment by derfloDev** [comment]
Yes, i have two of them laying aorund, because i cant get them working wirh homeassistant flawless. I hope it will be supported as a component by homeassistant soon. Unfortunately, I don't have time to keep working on it by myself. @amahlaka
